src/coreclr/jit/emitxarch.cpp

```diff
@@ -2168,7 +2915,7 @@ inline UNATIVE_OFFSET emitter::emitInsSizeRR(instrDesc* id, code_t code)
     instruction ins  = id->idIns();
     emitAttr    attr = id->idOpSize();
 
-    UNATIVE_OFFSET sz = emitGetAdjustedSize(ins, attr, code);
+    UNATIVE_OFFSET sz = emitGetAdjustedSizeSIMD(ins, attr, code);
```

Comment:
This name is potentially more confusing here since `RR` is not restricted to only SIMD instructions

Reply:
That's fair. Eventually we want to remove the existing emitGetAdjustedSize() and make this the new emitGetAdjustedSize().
So, maybe `emitGetAdjustedSizeWithEvex()` with a TODOAVX512 saying eventually make the change?

Reply:
emitGetAdjustedSizeEvexAware is fine by me - changed